install.rdf

```diff
@@ -5,20 +5,19 @@
     <em:bootstrap>true</em:bootstrap>
     <em:type>2</em:type>
     <em:unpack>false</em:unpack>
-    <em:version>1.0.0</em:version>
+    <em:version>0.1.2</em:version>
```

Comment:
FWIW on Test Pilot itself, we [update the manually-specified package version](https://github.com/mozilla/testpilot/blob/master/addon/bin/update-version) with a suffix of either "-dev-{git commit sha}" or "-tag-{release tag}" on build. Also, we have a [custom signing script](https://github.com/mozilla/testpilot/blob/master/addon/bin/sign) that will download the already-signed XPI on repeat attempts to build the same commit - which turns out to be handy in build automation.